.github/ISSUE_TEMPLATE/4_Documentation_issue.md

```diff
@@ -1,5 +1,5 @@
 ---
-name: ⛔ Documentation Issue
+name: 🗏 Documentation Issue
```

Comment:
This isn't rendering for me. It may not be one of the emojis supported by Github:

https://gist.github.com/rxaviers/7360908

Reply:
Maybe just 📖 ?

Reply:
ah yes, i thought all unicode is supported.

Reply:
I think i visually recognize :books: faster?

Reply:
hm, not sure when it's so small, maybe :green_book: ?

Reply:
Nah, I guess :book: is fine as well. Should I update the PR?

Reply:
Darn, the MD code breaks the template, so one has to copy in the emoji? So weird..

Reply:
wow, i'm unable to copy the icon in, the editor wipes it away immediately. Don't know what's going on.

Reply:
You could try using a separate editor and then pushing to the branch. Github and emojis are always strange.